stdlib/public/core/AssertCommon.swift

```diff
@@ -238,3 +238,51 @@ func _undefined<T>(
 ) -> T {
   _assertionFailure("Fatal error", message(), file: file, line: line, flags: 0)
 }
+
+/// Called when falling off the end of a switch and the type can be represented
+/// as a raw value.
+///
+/// This function should not be inlined because it is cold and inlining just
+/// bloats code. It doesn't take a source location because it's most important
+/// in release builds anyway (old apps that are run on new OSs).
+@inline(never)
+public // COMPILER_INTRINSIC
+func _diagnoseUnexpectedEnumCaseValue<RawValue>(type: Any.Type,
```

Comment:
The call site will be less code if you pass in a T.Type for a new generic parameter T

Reply:
Oh, interesting. I had assumed the opposite because there would be…more…parameters, no wait that doesn't make sense. Thanks!